mail/mail-utils.go

```diff
@@ -74,6 +78,20 @@ func setPolicyReservedBodyData(policy models.Policy, bodyData *BodyData) {
 	}
 }
 
+func getPolicyRenewDraftBodyData(policy models.Policy, hasMandate bool) BodyData {
+	priceGross := policy.PriceGross
+	if policy.PaymentSplit == string(models.PaySplitMonthly) {
+		priceGross = policy.PriceGrossMonthly
```

Comment:
my only concern here is for unformatted values like: 98,000000034
Should we use the data as string like:
```golang
value := policy.PriceGross
if policy.PaymenSplit == string(models.PaySplitMonthly) {
   value = polic.PriceGrossMonthly
}
priceGross := fmt.Sprintf("%.2f", value)
```
would need to update
```golang
struct BodyData {
   ...
   PriceGross string
}
```